Proposal for apps aggregation task (#73)
solid/team: tasks/apps-aggregation.md

> # Aggregation of Solid-interoperable apps

Reviewer: Perhaps we can simplify the task title and leave criteria-like details (which may be subject to a change) in the description.

```suggestion
# Manage Solid apps
```

Reviewer 2: ```suggestion
# Managing the list of Solid apps
```

> # Aggregation of Solid-interoperable apps

Reviewer: Alternative title if we combine this task with task to curating and maintaining list of tools ( https://github.com/solid/team/pull/74#pullrequestreview-1945595856 ):

```suggestion
# Manage list of Solid apps and tools
```

Author: I'll fix this soon

## Responsibilities

> Members should aggregate a list of apps that make use of the Solid specifications in some way.

Reviewer: ```suggestion
The task force is responsible for curating and maintaining a list of Solid applications.
```

> The apps should meet certain criteria that the task force makes publicly available. The task force needs to develop
> these criteria (or find old existing ones), ensuring that they're fair and maintainable.

Reviewer: ```suggestion
The list of Solid applications is made publicly accessible on the [solidproject.org](https://solidproject.org/) website.

The curation and maintanence of the list of Solid applications is based on the [inclusion and exclusion criteria](https://solidproject.org/apps#app-inclusion-and-exclusion-criteria). The task force is also responsible for processing proposed changes to the criteria.
```

Reviewer 2: Tweaking @csarven's suggestion
```suggestion
The list of Solid applications is made publicly accessible on the [solidproject.org](https://solidproject.org/) website.

Curation and maintenance of the list of Solid applications is based on the [inclusion and exclusion criteria](https://solidproject.org/apps#app-inclusion-and-exclusion-criteria). The task force is also responsible for processing proposed changes to the criteria.
```

> Submissions of new apps should also be made publicly, through mechanics such as Pull Requests to solidproject.org. 

Reviewer: ```suggestion
A record of requesting an update to the list of applications is made through a GitHub Pull Request on the [solid/solidproject.org](solid/solidproject.org) repository.
```

Reviewer 2: Starting from @csarven's suggestion...
```suggestion
Requesting an update to the list of applications is recorded through a GitHub Pull Request on the [solid/solidproject.org](https://github.com/solid/solidproject.org/pulls) repository.
```

> # Aggregation of Solid-compatible apps

Reviewer: I'm not sure I understand what "compatible" means or entails as opposed to saying "interoperable" or perhaps "conforming to Solid specifications" or something along those lines?

Author: English is not my first language, but I thought compatible was a good word to describe an app making use of technologies that are based on the Solid specification. But I can use interoperable or conforming instead, I have no strong feelings

## Responsibilities

Members should aggregate a list of Solid-compatible apps that meet certain criteria.

> The task force needs to develop these criteria, ensuring that they're fair and maintainable.

Author: Please let me know if there are some developed criteria already, that would be very useful for this group.

> The task force needs to develop these criteria, ensuring that they're fair and maintainable.

Author: I've added a paragraph on submissions.

> The task force needs to develop these criteria, ensuring that they're fair and maintainable.

Reviewer: Great, thanks! I've reviewed and included reference to criteria that we currently have on the website. Some of the content around the criteria could be cleaned up but we should reuse it as is. Leave the discussion to criteria as a separate matter if we need to revisit.

I've noted that the website refers to "Solid-compatible" so your original suggestion was already aligned with that. So, never mind my suggestion on the title and lets stick to covering what we currently have as is (for the most part).
